horace_core/sqw/PixelData/@PixelDataFileBacked/apply_c.m

```diff
@@ -0,0 +1,42 @@
+function sqw_out = apply_c(sqw_in,page_op)
```

Comment:
You want something funy. There is description on what the operation is doing in the function parameters. 
Together with description of page operation, its reasonably complete. 

For this you probably want some additional design document which does not exist in case of other algorithms. 
Why do you want additional design document for this algorithm and not for the dozen of others?